src/components/search/search.jsx

```diff
@@ -0,0 +1,172 @@
+import { cn } from '@/utilities/functions';
+import { File, Folder, Search, SquareSlash } from 'lucide-react';
+import { useState, useEffect } from 'react';
+import { Label } from '..';
+import Loader from '../loader';
+
+const SearchBox = ( props ) => {
+	const {
+		onChange = () => { },
+		handleResultClick = () => { },
+		placeholder = 'Search',
+		variant = 'primary', // primary, secondary, ghost
+		disabled = false,
+		size = 'sm', // sm, md, lg
+		loading = false,
+		loadingIcon = <Loader />, // Default loading icon
+		searchResult = [],
+		additionalResult = [],
+		searchResultIcon = <File />,
+		additionalResultLabel = 'Category',
```

Comment:
Custom text should not be placed. In this case set label if something is passed only.